src/Raven.Client/ServerWide/Operations/DeleteDatabasesOperation.cs

```diff
@@ -29,6 +29,21 @@ public DeleteDatabasesOperation(string databaseName, bool hardDelete, string fro
                 _parameters.FromNodes = new[] { fromNode };
         }
 
+        public DeleteDatabasesOperation(string databaseName, bool hardDelete, string fromNode, int shard, TimeSpan? timeToWaitForConfirmation = null)
```

Comment:
since this is 6.0, you can remove this ctor and add `int? shard = null` to the main one? Thoughts?
Also shard vs shardNumber, what it should be?

Reply:
I prefer the explicit one, and yes should be `shardNumber`

Reply:
In the explicit one there is no option to pass null in the node. To prevent the user from trying to remove an entire shard from all the nodes

Reply:
I'll add the null check for the node